browser/renderer_context_menu/brave_spelling_menu_observer_browsertest.cc

```diff
@@ -14,7 +14,7 @@
 #include "chrome/test/base/in_process_browser_test.h"
 #include "components/prefs/pref_service.h"
 #include "components/spellcheck/browser/pref_names.h"
-#include "content/public/common/context_menu_params.h"
+#include "content/public/browser/context_menu_params.h"
```

Comment:
deps change? content/public/common still needed? content/public/browser need to be added?

Reply:
We don't have a direct dep on content/public/common, both are deps in //components/renderer_context_menu